models/index.js

```diff
@@ -35,18 +35,20 @@ async function createDatabase(options) {
   const db = {};
   db.sequelize = options;
 
-  fs.readdirSync(__dirname)
-    .filter((file) => file.indexOf(".") !== 0 && file !== basename && file.slice(-3) === ".js")
-    .forEach(async (file) => {
+  const files = fs
+    .readdirSync(__dirname)
+    .filter((file) => file.indexOf(".") !== 0 && file !== basename && file.slice(-3) === ".js");
+
+  const models = await Promise.all(
+    files.map(async (file) => {
       const model = (await import(path.join(__dirname, file))).default(options, Sequelize);
-      db[model.name] = model;
-    });
-
-  Object.keys(db).forEach((modelName) => {
-    if (db[modelName].associate) {
-      db[modelName].associate(db);
-    }
-  });
+      return model;
+    }),
+  );
+
+  for (const model of models) {
+    db[model.name] = model;
+  }
 
```

Comment:
Did you mean to delete this part? :sweat_smile:
```suggestion
  Object.keys(db).forEach((modelName) => {
    if (db[modelName].associate) {
      db[modelName].associate(db);
    }
  });
```

Reply:
I forgot to mention adding await to this line below as well
```const adminDb = await createDatabase(sequelize);```